src/Map/src/Twig/MapRuntime.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Twig;
+
+use Symfony\UX\Map\Map;
+use Symfony\UX\Map\MapRegistry;
+use Symfony\UX\StimulusBundle\Helper\StimulusHelper;
+use Twig\Extension\RuntimeExtensionInterface;
+
+/**
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+final class MapRuntime implements RuntimeExtensionInterface
+{
+    public function __construct(
+        private readonly StimulusHelper $stimulus,
+        private readonly MapRegistry $mapRegistry,
+    ) {
+    }
+
+    public function renderScriptTags(): string
+    {
+        if (!$maps = $this->mapRegistry->all()) {
+            return '';
+        }
+
+        $scriptTags = [];
+
+        $jsConfig = ['providers' => []];
+        foreach ($maps as $map) {
+            $jsConfig['providers'][$map->getProvider()->getName()] = (object) $map->getProvider()->getOptions();
+        }
+        $scriptTags[] = \sprintf('<script>window.__symfony_ux_maps = %s</script>', json_encode($jsConfig, flags: \JSON_THROW_ON_ERROR));
```

Comment:
Remind me what is the goal of this ? 

A leaflet map does not require any of this as it can be done map per map right ?

Reply:
Indeed, for Leaflet this is not necessary, but for Google Maps and surely other providers, their API is globally loaded and passing options like this is nice to me.

If we do it on the rendered Map (`<div data-controller="...">`), I'm afraid of potential conflicts (thus, it was especially true before when having multiples maps with different providers of the same type), or _data waste_ (if we have 10 maps, should we pass the same configuration for GoogleMaps 10 times?)

Reply:
https://developers.google.com/maps/documentation/javascript/google-loader-migration#dynamic_loading_from_another_javascript_file

Reply:
That's what the `@googlemaps/js-api-loader` package does, but I was maily speaking about:
- possible race-conditions between multiples GoogleMaps initialization at the same time, 
- and having the provider configuration in `<div data-controller>` multiple times (e.g.: `{"language":"fr","region":"FR","retries":10,"version":"weekly","apiKey":"MY_API_KEY"}}`

Reply:
Fixed